From f9824af8936f61b424da3ac3eba8937fa3250042 Mon Sep 17 00:00:00 2001
From: Soren Friis <soren.friis@intel.com>
Date: Fri, 10 Feb 2017 14:16:28 +0200
Subject: [PATCH] Don't use adaptive 8x8 DCT

When the adaptive 8x8 DCT transform is enabled for the
H.264 video encoder, the bitstream encoded for 720p
resolution is somehow corrupted, which prevent the
playback of it using gst-play-1.0.

Turn off this encoding tool until the corruption
has been corrected.

Signed-off-by: Soren Friis <soren.friis@intel.com>
---
 gst-libs/gst/vaapi/gstvaapiencoder_h264.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst-libs/gst/vaapi/gstvaapiencoder_h264.c b/gst-libs/gst/vaapi/gstvaapiencoder_h264.c
index 49a7988..e074498 100644
--- a/gst-libs/gst/vaapi/gstvaapiencoder_h264.c
+++ b/gst-libs/gst/vaapi/gstvaapiencoder_h264.c
@@ -3102,7 +3102,7 @@ gst_vaapi_encoder_h264_get_default_properties (void)
       g_param_spec_boolean ("dct8x8",
           "Enable 8x8 DCT",
           "Enable adaptive use of 8x8 transforms in I-frames",
-          TRUE, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+          FALSE, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
   /**
    * GstVaapiEncoderH264:cpb-length:
-- 
2.7.4

